### ISSUE TITLE
2025-04-30, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#155]
# Today's Arxiv Papers

(1) [Toward Efficient Exploration by Large Language Model Agents](http://arxiv.org/abs/2504.20997v1), 2025-04-29

(2) [X-Fusion: Introducing New Modality to Frozen Large Language Models](http://arxiv.org/abs/2504.20996v1), 2025-04-29

(3) [ACE: A Security Architecture for LLM-Integrated App Systems](http://arxiv.org/abs/2504.20984v1), 2025-04-29

(4) [Jekyll-and-Hyde Tipping Point in an AI's Behavior](http://arxiv.org/abs/2504.20980v1), 2025-04-29

(5) [Real-Time Wayfinding Assistant for Blind and Low-Vision Users](http://arxiv.org/abs/2504.20976v1), 2025-04-29

(6) [SetKE: Knowledge Editing for Knowledge Elements Overlap](http://arxiv.org/abs/2504.20972v1), 2025-04-29

(7) [AegisLLM: Scaling Agentic Systems for Self-Reflective Defense in LLM Security](http://arxiv.org/abs/2504.20965v1), 2025-04-29

(8) [OSVBench: Benchmarking LLMs on Specification Generation Tasks for Operating System Verification](http://arxiv.org/abs/2504.20964v1), 2025-04-29

(9) [Information Gravity: A Field-Theoretic Model for Token Selection in Large Language Models](http://dx.doi.org/10.5281/zenodo.15289890), 2025-04-29

(10) [Trace-of-Thought: Enhanced Arithmetic Problem Solving via Reasoning Distillation From Large to Small Language Models](http://arxiv.org/abs/2504.20946v1), 2025-04-29

(11) [ChestX-Reasoner: Advancing Radiology Foundation Models with Reasoning through Step-by-Step Verification](http://arxiv.org/abs/2504.20930v1), 2025-04-29

(12) [DYNAMAX: Dynamic computing for Transformers and Mamba based architectures](http://arxiv.org/abs/2504.20922v1), 2025-04-29

(13) [An Empirical Study on the Capability of LLMs in Decomposing Bug Reports](http://arxiv.org/abs/2504.20911v1), 2025-04-29

(14) [Classifier-to-Bias: Toward Unsupervised Automatic Bias Detection for Visual Classifiers](http://arxiv.org/abs/2504.20902v1), 2025-04-29

(15) [LELANTE: LEveraging LLM for Automated ANdroid TEsting](http://arxiv.org/abs/2504.20896v1), 2025-04-29

(16) [The Leaderboard Illusion](http://arxiv.org/abs/2504.20879v1), 2025-04-29

(17) [JaccDiv: A Metric and Benchmark for Quantifying Diversity of Generated Marketing Text in the Music Industry](http://arxiv.org/abs/2504.20849v1), 2025-04-29

(18) [Enhancing Non-Core Language Instruction-Following in Speech LLMs via Semi-Implicit Cross-Lingual CoT Reasoning](http://arxiv.org/abs/2504.20835v1), 2025-04-29

(19) [Reinforcement Learning for LLM Reasoning Under Memory Constraints](http://arxiv.org/abs/2504.20834v1), 2025-04-29

(20) [Ascendra: Dynamic Request Prioritization for Efficient LLM Serving](http://arxiv.org/abs/2504.20828v1), 2025-04-29

(21) [Secure Coding with AI, From Creation to Inspection](http://arxiv.org/abs/2504.20814v1), 2025-04-29

(22) [Unlocking User-oriented Pages: Intention-driven Black-box Scanner for Real-world Web Applications](http://arxiv.org/abs/2504.20801v1), 2025-04-29

(23) [Hallucination by Code Generation LLMs: Taxonomy, Benchmarks, Mitigation, and Challenges](http://arxiv.org/abs/2504.20799v1), 2025-04-29

(24) [Q-Fusion: Diffusing Quantum Circuits](http://arxiv.org/abs/2504.20794v1), 2025-04-29

(25) [Using LLMs in Generating Design Rationale for Software Architecture Decisions](http://arxiv.org/abs/2504.20781v1), 2025-04-29

(26) [Turing Machine Evaluation for Large Language Model](http://arxiv.org/abs/2504.20771v1), 2025-04-29

(27) [Chain-of-Defensive-Thought: Structured Reasoning Elicits Robustness in Large Language Models against Reference Corruption](http://arxiv.org/abs/2504.20769v1), 2025-04-29

(28) [Understanding Large Language Model Supply Chain: Structure, Domain, and Vulnerabilities](http://arxiv.org/abs/2504.20763v1), 2025-04-29

(29) [Beyond the Last Answer: Your Reasoning Trace Uncovers More than You Think](http://arxiv.org/abs/2504.20708v1), 2025-04-29

(30) [Can LLMs Detect Intrinsic Hallucinations in Paraphrasing and Machine Translation?](http://arxiv.org/abs/2504.20699v1), 2025-04-29

(31) [Identifying Uncertainty in Self-Adaptive Robotics with Large Language Models](http://arxiv.org/abs/2504.20684v1), 2025-04-29

(32) [CoCo-Bench: A Comprehensive Code Benchmark For Multi-task Large Language Model Evaluation](http://arxiv.org/abs/2504.20673v1), 2025-04-29

(33) [A Generative-AI-Driven Claim Retrieval System Capable of Detecting and Retrieving Claims from Social Media Platforms in Multiple Languages](http://arxiv.org/abs/2504.20668v1), 2025-04-29

(34) [ComplexVCoder: An LLM-Driven Framework for Systematic Generation of Complex Verilog Code](http://arxiv.org/abs/2504.20653v1), 2025-04-29

(35) [Combatting Dimensional Collapse in LLM Pre-Training Data via Diversified File Selection](http://arxiv.org/abs/2504.20644v1), 2025-04-29

(36) [Cooking Up Creativity: A Cognitively-Inspired Approach for Enhancing LLM Creativity through Structured Representations](http://arxiv.org/abs/2504.20643v1), 2025-04-29

(37) [Cognitive maps are generative programs](http://arxiv.org/abs/2504.20628v1), 2025-04-29

(38) [PaRT: Enhancing Proactive Social Chatbots with Personalized Real-Time Retrieval](http://arxiv.org/abs/2504.20624v1), 2025-04-29

(39) [The Hidden Risks of LLM-Generated Web Application Code: A Security-Centric Evaluation of Code Generation Capabilities in Large Language Models](http://arxiv.org/abs/2504.20612v1), 2025-04-29

(40) [Information Retrieval in the Age of Generative AI: The RGB Model](http://arxiv.org/abs/2504.20610v1), 2025-04-29

(41) [WenyanGPT: A Large Language Model for Classical Chinese Tasks](http://arxiv.org/abs/2504.20609v1), 2025-04-29

(42) [ReasonIR: Training Retrievers for Reasoning Tasks](http://arxiv.org/abs/2504.20595v1), 2025-04-29

(43) [Reinforcement Learning for Reasoning in Large Language Models with One Training Example](http://arxiv.org/abs/2504.20571v1), 2025-04-29

(44) [ReCIT: Reconstructing Full Private Data from Gradient in Parameter-Efficient Fine-Tuning of Large Language Models](http://arxiv.org/abs/2504.20570v1), 2025-04-29

(45) [BrAIcht, a theatrical agent that speaks like Bertolt Brecht's characters](http://arxiv.org/abs/2504.20552v1), 2025-04-29

(46) [Revisiting the MIMIC-IV Benchmark: Experiments Using Language Models for Electronic Health Records](http://arxiv.org/abs/2504.20547v1), 2025-04-29

(47) [Conversations with AI Chatbots Increase Short-Term Vaccine Intentions But Do Not Outperform Standard Public Health Messaging](http://arxiv.org/abs/2504.20519v1), 2025-04-29

(48) [MuRAL: A Multi-Resident Ambient Sensor Dataset Annotated with Natural Language for Activities of Daily Living](http://arxiv.org/abs/2504.20505v1), 2025-04-29

(49) [UniDetox: Universal Detoxification of Large Language Models via Dataset Distillation](http://arxiv.org/abs/2504.20500v1), 2025-04-29

(50) [Token-Efficient Prompt Injection Attack: Provoking Cessation in LLM Reasoning via Adaptive Token Compression](http://arxiv.org/abs/2504.20493v1), 2025-04-29

(51) [Enhancing LLM Language Adaption through Cross-lingual In-Context Pre-training](http://arxiv.org/abs/2504.20484v1), 2025-04-29

(52) [Robustness via Referencing: Defending against Prompt Injection Attacks by Referencing the Executed Instruction](http://arxiv.org/abs/2504.20472v1), 2025-04-29

(53) [Fane at SemEval-2025 Task 10: Zero-Shot Entity Framing with Large Language Models](http://arxiv.org/abs/2504.20469v1), 2025-04-29

(54) [A Summary on GUI Agents with Foundation Models Enhanced by Reinforcement Learning](http://arxiv.org/abs/2504.20464v1), 2025-04-29

(55) [TAMO:Fine-Grained Root Cause Analysis via Tool-Assisted LLM Agent with Multi-Modality Observation Data](http://arxiv.org/abs/2504.20462v1), 2025-04-29

(56) [SAS-Prompt: Large Language Models as Numerical Optimizers for Robot Self-Improvement](http://arxiv.org/abs/2504.20459v1), 2025-04-29

(57) [Enhancing News Recommendation with Hierarchical LLM Prompting](http://arxiv.org/abs/2504.20452v1), 2025-04-29

(58) [Team ACK at SemEval-2025 Task 2: Beyond Word-for-Word Machine Translation for English-Korean Pairs](http://arxiv.org/abs/2504.20451v1), 2025-04-29

(59) [On Psychology of AI -- Does Primacy Effect Affect ChatGPT and Other LLMs?](http://arxiv.org/abs/2504.20444v1), 2025-04-29

(60) [GaLore 2: Large-Scale LLM Pre-Training by Gradient Low-Rank Projection](http://arxiv.org/abs/2504.20437v1), 2025-04-29

(61) [RV-Syn: Rational and Verifiable Mathematical Reasoning Data Synthesis based on Structured Function Library](http://arxiv.org/abs/2504.20426v1), 2025-04-29

(62) [Plant Disease Detection through Multimodal Large Language Models and Convolutional Neural Networks](http://arxiv.org/abs/2504.20419v1), 2025-04-29

(63) [Enhancing Leakage Attacks on Searchable Symmetric Encryption Using LLM-Based Synthetic Data Generation](http://arxiv.org/abs/2504.20414v1), 2025-04-29

(64) [CrashFixer: A crash resolution agent for the Linux kernel](http://arxiv.org/abs/2504.20412v1), 2025-04-29

(65) [Skill Discovery for Software Scripting Automation via Offline Simulations with LLMs](http://arxiv.org/abs/2504.20406v1), 2025-04-29

(66) [FiLA-Video: Spatio-Temporal Compression for Fine-Grained Long Video Understanding](http://arxiv.org/abs/2504.20384v1), 2025-04-29

(67) [An Empirical Study on Common Defects in Modern Web Browsers Using Knowledge Embedding in GPT-4o](http://arxiv.org/abs/2504.20381v1), 2025-04-29

(68) [DMDTEval: An Evaluation and Analysis of LLMs on Disambiguation in Multi-domain Translation](http://arxiv.org/abs/2504.20371v1), 2025-04-29

(69) [Thoughtful, Confused, or Untrustworthy: How Text Presentation Influences Perceptions of AI Writing Tools](http://dx.doi.org/10.1145/3698061.3726907), 2025-04-29

(70) [Local Prompt Optimization](http://arxiv.org/abs/2504.20355v1), 2025-04-29

(71) [CarbonCall: Sustainability-Aware Function Calling for Large Language Models on Edge Devices](http://arxiv.org/abs/2504.20348v1), 2025-04-29

(72) ["I've talked to ChatGPT about my issues last night.": Examining Mental Health Conversations with Large Language Models through Reddit Analysis](http://arxiv.org/abs/2504.20320v1), 2025-04-29

(73) [Enhancing Systematic Reviews with Large Language Models: Using GPT-4 and Kimi](http://arxiv.org/abs/2504.20276v1), 2025-04-28

(74) [Can Large Language Models Learn Formal Logic? A Data-Driven Training and Evaluation Framework](http://arxiv.org/abs/2504.20213v1), 2025-04-28

(75) [Prompting LLMs for Code Editing: Struggles and Remedies](http://arxiv.org/abs/2504.20196v1), 2025-04-28

(76) [BLADE: Benchmark suite for LLM-driven Automated Design and Evolution of iterative optimisation heuristics](http://arxiv.org/abs/2504.20183v1), 2025-04-28

(77) [Toward Evaluative Thinking: Meta Policy Optimization with Evolving Reward Models](http://arxiv.org/abs/2504.20157v1), 2025-04-28

(78) [AutoJudge: Judge Decoding Without Manual Annotation](http://arxiv.org/abs/2504.20039v1), 2025-04-28

(79) [SpatialReasoner: Towards Explicit and Generalizable 3D Spatial Reasoning](http://arxiv.org/abs/2504.20024v1), 2025-04-28

(80) [Better To Ask in English? Evaluating Factual Accuracy of Multilingual LLMs in English and Low-Resource Languages](http://arxiv.org/abs/2504.20022v1), 2025-04-28

(81) [Modular Machine Learning: An Indispensable Path towards New-Generation Large Language Models](http://arxiv.org/abs/2504.20020v1), 2025-04-28

(82) [Applying LLM-Powered Virtual Humans to Child Interviews in Child-Centered Design](http://dx.doi.org/10.1145/3713043.3731551), 2025-04-28

(83) [LLM-Generated Fake News Induces Truth Decay in News Ecosystem: A Case Study on Neural News Recommendation](http://dx.doi.org/10.1145/3726302.3730027), 2025-04-28

(84) [Towards Automated Scoping of AI for Social Good Projects](http://arxiv.org/abs/2504.20010v1), 2025-04-28

(85) [Chatbot Arena Meets Nuggets: Towards Explanations and Diagnostics in the Evaluation of LLM Responses](http://arxiv.org/abs/2504.20006v1), 2025-04-28

(86) [Knowledge Distillation of Domain-adapted LLMs for Question-Answering in Telecom](http://arxiv.org/abs/2504.20000v1), 2025-04-28

(87) [TD-EVAL: Revisiting Task-Oriented Dialogue Evaluation by Combining Turn-Level Precision with Dialogue-Level Comparisons](http://arxiv.org/abs/2504.19982v1), 2025-04-28

(88) [Accurate and Diverse LLM Mathematical Reasoning via Automated PRM-Guided GFlowNets](http://arxiv.org/abs/2504.19981v1), 2025-04-28

(89) [From Concept to Practice: an Automated LLM-aided UVM Machine for RTL Verification](http://arxiv.org/abs/2504.19959v2), 2025-04-28

(90) [Securing Agentic AI: A Comprehensive Threat Model and Mitigation Framework for Generative AI Agents](http://arxiv.org/abs/2504.19956v1), 2025-04-28

(91) [Enhancing Surgical Documentation through Multimodal Visual-Temporal Transformers and Generative AI](http://arxiv.org/abs/2504.19918v1), 2025-04-28

(92) [Can AI Agents Design and Implement Drug Discovery Pipelines?](http://arxiv.org/abs/2504.19912v1), 2025-04-28

(93) [GenCLS++: Pushing the Boundaries of Generative Classification in LLMs Through Comprehensive SFT and RL Studies Across Diverse Datasets](http://arxiv.org/abs/2504.19898v1), 2025-04-28

(94) [CineVerse: Consistent Keyframe Synthesis for Cinematic Scene Composition](http://arxiv.org/abs/2504.19894v1), 2025-04-28

(95) [semi-PD: Towards Efficient LLM Serving via Phase-Wise Disaggregated Computation and Unified Storage](http://arxiv.org/abs/2504.19867v1), 2025-04-28

(96) [CoherenDream: Boosting Holistic Text Coherence in 3D Generation via Multimodal Large Language Models Feedback](http://arxiv.org/abs/2504.19860v1), 2025-04-28

(97) [The Automation Advantage in AI Red Teaming](http://arxiv.org/abs/2504.19855v2), 2025-04-28

(98) [LLM-Powered GUI Agents in Phone Automation: Surveying Progress and Prospects](http://arxiv.org/abs/2504.19838v1), 2025-04-28

(99) [PhenoAssistant: A Conversational Multi-Agent AI System for Automated Plant Phenotyping](http://arxiv.org/abs/2504.19818v1), 2025-04-28

(100) [Can a Crow Hatch a Falcon? Lineage Matters in Predicting Large Language Model Performance](http://arxiv.org/abs/2504.19811v1), 2025-04-28
